vscode-car-plugin/src/main/java/org/wso2/maven/datamapper/DataMapperBundler.java

```diff
@@ -73,7 +73,7 @@ public DataMapperBundler(CARMojo mojoInstance, String resourcesDirectory) {
      * @throws MojoExecutionException if an error occurs while executing the Maven invoker.
      */
     public void bundleDataMapper() throws DataMapperException, MojoExecutionException {
-        String dataMapperDirectoryPath = resourcesDirectory + File.separator + Constants.DATA_MAPPER_DIR_PATH;
+        String dataMapperDirectoryPath = resourcesDirectory + File.separator + Constants.DATA_MAPPER_DIR_NAME;
```

Comment:
Wont this break for the existing users who currently have the relevant files in the gov directory?

Reply:
Updated with capturing data mappers in gov directory and the new directory.